event.go

```diff
@@ -0,0 +1,52 @@
+package basictracer
+
+import "github.com/opentracing/opentracing-go"
+
+// An Event is emitted when a mutating command is called on a Span.
+type Event interface{}
+
+// EventCreate is emitted when a Span is created.
+type EventCreate struct{ OperationName string }
```

Comment:
Now that I think of it, this isn't as useful as I thought. Since the `EventFinish` is the `RawSpan`, you can access all of that attributes of the `Span` like the `StartTime`. Still though, it's food for thought.